internal/pkg/template/templates/environment/cf.yml

```diff
@@ -390,6 +470,31 @@ Outputs:
     Value: !Ref DefaultHTTPTargetGroup
     Export:
       Name: !Sub ${AWS::StackName}-DefaultHTTPTargetGroup
+  InternalLoadBalancerDNSName:
```

Comment:
Would you mind checking which one of these outputs are we using in our codebase?

For example, I don't think service stacks need the `DefaultInternalALBHTTPTargetGroup` 🤔